tests/org.eclipse.n4js.hlc.tests/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,7 @@ Require-Bundle: org.eclipse.n4js,
  org.eclipse.n4js.hlc,
  org.eclipse.n4js.hlc.base,
  org.junit,
- com.google.guava;bundle-version="15.0.0",
+ com.google.guava,
```

Comment:
don't we need version in at least one place for Oomph to properly generate target platform file?

Reply:
The generated Guava version is 18